MaterialDesignThemes.Wpf/TabItemAssist.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Windows;
+
+namespace MaterialDesignThemes.Wpf
+{
+    public static class TabItemAssist
```

Comment:
Is this being used anywhere?
I can see the Leading/Top icon in the [demo](https://material.io/components/tabs#usage) but I would expect that if someone wanted that look, they would simply put a `StackPanel` inside of the `TabItem.Header` and setup the layout that way.

Reply:
This isn't be used. I commited this a bit too soon, I'll remove the class in a following commit.

I was thinking this could be a preferred way of doing this, analogous with how Button has a LeadingIcon.